easybuild/easyconfigs/g/GROMACS/GROMACS-2021.5-foss-2021b-PLUMED-2.8.0.eb

```diff
@@ -9,25 +9,27 @@
 # * Kenneth Hoste <kenneth.hoste@ugent.be>
 # * Adam Huffman <adam.huffman@crick.ac.uk>
 # * Ake Sandgren <ake.sandgren@hpc2n.umu.se>
-# * J. Sassmannshausen <Crick HPC team>
 # * Dugan Witherick <dugan.witherick@warwick.ac.uk>
 # * Christoph Siegert <christoph.siegert@uni-leipzig.de>
+# * J. Sassmannshausen <j.sassmannshausen@imperial.ac.uk>
 # License::   MIT/GPL
 
 name = 'GROMACS'
 version = '2021.5'
-versionsuffix = '-CUDA-%(cudaver)s'
+_plumedver = '2.8.0'
+versionsuffix = '-PLUMED-%s' % _plumedver
```

Comment:
@sassy-crick You're still changing an existing easyconfig here, which we shouldn't do.

Can you add back `GROMACS-2021.5-foss-2021b-CUDA-11.4.1.eb` from `develop` in your PR branch?

Reply:
@sassy-crick I have added `GROMACS-2021.5-foss-2021b-CUDA-11.4.1.eb` in https://github.com/sassy-crick/easybuild-easyconfigs/pull/8

Reply:
Sorry for the delay in getting back here. Is that now all sorted or are there still problems?

Reply:
Thanks a lot! The files look good now :+1: 

I will asked the bot for the test report.